lib/metadata/linux/LinuxUtils.rb

```diff
@@ -193,5 +214,52 @@ def self.parse_openstack_status(lines)
         }
       end
     end
+
+    def self.parse_openstack_container_status(lines)
```

Comment:
As Rubocop mentioned, this method is pretty hard to read with all the chained blocks and conditionals. It should be broken down so that it's easier to follow.